### PR TITLE
Update about.md for Nextercism

### PR DESCRIPTION
**Pull request by rpottsoh**
First crack at updating the about.md.  Essentially just whittled down the original file.

### CODE REVIEW

**Comment by rpottsoh** [comment]
@kotp or @kytrinyx I wouldn't mind a critique of this.

**Comment by kytrinyx** [comment]
My first thought on reading through this is that it's lots and lots of facts, and I have a hard time hooking into it emotionally.

Rather than critiquing this directly, I'd like to ask you a few questions about your experience with Delphi, to see if I can tease out some ideas about direction. None of these answers are meant to be things to put into the ABOUT document, it's just a starting point to think about how to tweak it.

* What made you start learning Delphi in the first place?
* How is it different from other languages that you like?
* How is it different from other languages that frustrate you?
* What's the most fun problem you've solved with Delphi?
* What do you like the most about Delphi?
* What are some of your favorite things in the language itself?
* What are some of your favorite projects written with Delphi?

And then a more historical question: do you know what motivated the authors of the language to create it?

For example For example, with Ruby, the creator wanted to make something that makes programmers happy. He took all his favorite parts of his favorite languages. His motivation was _delight_.

With Go, on the other hand, the problem was the immense complexity and long compile times of C++. The authors wanted something that was far simpler, but still had the power and performance of C.

**Comment by rpottsoh** [comment]
@kytrinyx thanks for the feedback and ideas.  I'll work on this some more.

**Comment by kytrinyx** [comment]
I'd love to hear your answers to these questions—it would help me ask more and better questions.

**Comment by rpottsoh** [comment]
>What made you start learning Delphi in the first place?

Short answer, work.  Where I work utilizes the Pascal language.  I had "computer lessons" as a kid.  I was taught Applesoft BASIC.  When the instructor ran out of things to cover we started working on assignments in Pascal; early and mid 80's time frame.

>How is it different from other languages that you like?

Pascal tends to be more descriptive than most languages which makes it easier to read.  It isn't full of shorthand notations that you come across in other languages.

>How is it different from other languages that frustrate you?

Other languages that, to me, are to cryptic and hard to follow if you don't already have an eye for it.

>What's the most fun problem you've solved with Delphi?

The first thing that pops to mind is the Windows Installer.  I have had to figure out how to write a REST client and update the PATH.  I have never had a reason to do those two things and I enjoyed figuring it out.  I have also been working on a GUI problem for the Delphi track.  It is looking like it will end up being two problems.  The first that will focus on the business logic of the app the student will be writing and the second will be the GUI app that will implement the solution from the first problem.

>What do you like the most about Delphi?

It allows me to spin up UIs very quickly.  The software I write for work is used in machine control and data processing.  I have to design and code user interfaces that are intuitive to use and present information to the operator in as easy to comprehend a way as possible.

>What are some of your favorite things in the language itself?

Simplicity, expressiveness and readability, great for learning and for professional development.  While being an OOP language it hasn't lost the ability to be procedural as well 

>What are some of your favorite projects written with Delphi?

I decided to perform a search of the internet to see what famous software is written with Delphi.  It is quite a [list](https://jonlennartaasenden.wordpress.com/2014/11/06/famous-software-made-with-delphi/).  There are a few on the list that I use (or have used over the years), but frankly had no idea were developed with Delphi, so cool!  Skype, Nero Burning ROM, WinRAR, Partition Magic, BeyondCompare, Help and Manual, and recently I learned that the Niagara Falls lighting system software was developed with Delphi.

>And then a more historical question: do you know what motivated the authors of the language to create it?

I don't know what motivated them other than financial gain.  I have a good eBook written and published by Marco Cantu that gives some good background history.  I have found a lot of that history independently stated [here](https://www.thoughtco.com/history-of-delphi-1056847) by Zarco Gajic.  I know that the primary architect of Turbo Pascal and later Delphi (up to version 3) was Anders Hejlsberg, who later left Borland for Microsoft where he developed C#.

**Comment by kytrinyx** [comment]
@rpottsoh This is great. It feels like the real power of Delphi for you is in the UI aspects. I need to think on this a bit. I really appreciate the insight.

File: docs/ABOUT.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
 
-Delphi was originally developed by [Borland](https://en.wikipedia.org/wiki/Borland) as a [rapid application development](https://en.wikipedia.org/wiki/Rapid_application_development) tool for Windows as the successor of [Turbo Pascal](https://en.wikipedia.org/wiki/Turbo_Pascal). Delphi added full object-orientation to the existing language, and since then the language has grown and supports many other modern language features, including generics and anonymous methods, as well as unusual features such as inbuilt string types and native COM support.
+Delphi was originally developed by [Borland](https://en.wikipedia.org/wiki/Borland), lead by Anders Heilsberg as a [RAD](https://en.wikipedia.org/wiki/Rapid_application_development) tool for Windows as the successor of [Turbo Pascal](https://en.wikipedia.org/wiki/Turbo_Pascal). Delphi added full object-orientation to the existing language, and since then the language has grown and supports many other modern language features, including:
+  
+- **Generics**  
+- **Anonymous Methods (closures)**  
+- **inbuilt string types**  
+- **native COM support**
```
Comment (kotp) [inline docs/ABOUT.md]: Seems like the pattern for capitalization is not decided for these.  Should it be a capital letter for each first letter of the headers, or each word in each header?

File: docs/ABOUT.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+Delphi was originally developed by [Borland](https://en.wikipedia.org/wiki/Borland), lead by Anders Heilsberg as a [RAD](https://en.wikipedia.org/wiki/Rapid_application_development) tool for Windows as the successor of [Turbo Pascal](https://en.wikipedia.org/wiki/Turbo_Pascal). Delphi added full object-orientation to the existing language, and since then the language has grown and supports many other modern language features, including:
+  
+- **Generics**  
+- **Anonymous Methods (closures)**  
+- **inbuilt string types**  
+- **native COM support**  
+
+Like its predecessor Turbo Pascal, at the core of Delphi is a native code compiler.  Being able to quickly develop quickly executing software is something that has been synonymous with Turbo Pascal and now Delphi.  More recent versions of Delphi are able to compile native code for many different platforms:
```
Comment (kotp) [inline docs/ABOUT.md]: Consider the repeated word 'quickly', and if another adjective can be used.  Perhaps `efficiently` `fast`.

**Comment by rpottsoh** [comment]
:tada:  thanks @kotp
